contracts/FiatToken.cdc

```diff
@@ -1,1017 +1,1000 @@
-    pub let VaultUUIDPubPath: PublicPath
-    pub let VaultReceiverPubPath: PublicPath
-
-    pub let BlocklistExecutorStoragePath: StoragePath
-
-    pub let BlocklisterStoragePath: StoragePath
-    pub let BlocklisterCapReceiverPubPath: PublicPath
-    pub let BlocklisterUUIDPubPath: PublicPath
-    pub let BlocklisterPubSigner: PublicPath
-
-    pub let PauseExecutorStoragePath: StoragePath
-
-    pub let PauserStoragePath: StoragePath
-    pub let PauserCapReceiverPubPath: PublicPath
-    pub let PauserUUIDPubPath: PublicPath
-    pub let PauserPubSigner: PublicPath
-
-    pub let AdminExecutorStoragePath: StoragePath
-
-    pub let AdminStoragePath: StoragePath
-    pub let AdminCapReceiverPubPath: PublicPath
-    pub let AdminUUIDPubPath: PublicPath
-    pub let AdminPubSigner: PublicPath
-
-    pub let OwnerExecutorStoragePath: StoragePath
-
-    pub let OwnerStoragePath: StoragePath
-    pub let OwnerCapReceiverPubPath: PublicPath
-    pub let OwnerUUIDPubPath: PublicPath
-    pub let OwnerPubSigner: PublicPath
-
-    pub let MasterMinterExecutorStoragePath: StoragePath
-
-    pub let MasterMinterStoragePath: StoragePath
-    pub let MasterMinterCapReceiverPubPath: PublicPath
-    pub let MasterMinterUUIDPubPath: PublicPath
-    pub let MasterMinterPubSigner: PublicPath
-
-    pub let MinterControllerStoragePath: StoragePath
-    pub let MinterControllerUUIDPubPath: PublicPath
-    pub let MinterControllerPubSigner: PublicPath
-
-    pub let MinterStoragePath: StoragePath
-    pub let MinterUUIDPubPath: PublicPath
-
-
-    // ------- FiatToken States / Variables -------
-
-    pub let name: String
-    pub var version: String
-    // Set to true if the contract is paused
-    pub var paused: Bool
-    // The token total supply
-    pub var totalSupply: UFix64
-    // Blocked resources dictionary {resourceId: Block Height}
-    access(contract) let blocklist: {UInt64: UInt64}
-    // Managed minters dictionary {MinterController: Minter}
-    access(contract) let managedMinters: {UInt64: UInt64}
-    // Minter allowance dictionary {Minter: Allowance}
-    access(contract) let minterAllowances: { UInt64: UFix64}
-    
-
-    // ------- FiatToken Interfaces  -------
-
-    pub resource interface ResourceId {
-        pub fun UUID(): UInt64
-    }
-
-    pub resource interface AdminCapReceiver {
-        pub fun setAdminCap(cap: Capability<&AdminExecutor>)
-    }
-
-    pub resource interface OwnerCapReceiver {
-        pub fun setOwnerCap(cap: Capability<&OwnerExecutor>)
-    }
-
-    pub resource interface MasterMinterCapReceiver {
-        pub fun setMasterMinterCap(cap: Capability<&MasterMinterExecutor>)
-    }
-
-    pub resource interface BlocklisterCapReceiver {
-        pub fun setBlocklistCap(cap: Capability<&BlocklistExecutor>)
-    }
-
-    pub resource interface PauseCapReceiver {
-        pub fun setPauseCap(cap: Capability<&PauseExecutor>)
-    }
-
-    
-    // ------- Path linking -------
```

Comment:
I got rid of these functions because they were only each being used in one place, so I just put the capability issuing logic in the functions where these were being called from